<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detailsdisplay</class>
 <widget class="QDialog" name="detailsdisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sell Details</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #bfe0f4;</string>
  </property>
  <widget class="QGroupBox" name="groupBox_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>591</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(85, 85, 255);</string>
   </property>
   <property name="title">
    <string>Customer Idenity</string>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>581</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>600</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>540</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
    </property>
    <property name="text">
     <string>Display Total Sell Info</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_search">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);

border-radius:15px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_search">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reset">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>540</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
    </property>
    <property name="text">
     <string>Reset Table</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>60</y>
     <width>611</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(85, 85, 255);</string>
   </property>
   <property name="title">
    <string>Items Sold</string>
   </property>
   <widget class="QTableView" name="tableView_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>581</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>550</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
    </property>
    <property name="text">
     <string>Display Item Sell Info</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_search2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);

border-radius:15px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_search2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reset_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>550</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
    </property>
    <property name="text">
     <string>Reset Table</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
